### PR TITLE
[Torch] Update metrics for resnet18_imagenet_binarization_xnor

### PR DESCRIPTION
**Pull request by daniil-lyakhov**
### Changes

Metrics for resnet18_imagenet_binarization_xnor are updated

### Reason for changes

Green e2e_pytorch

### Tests

e2e_pytorch_2.0.1/18/

### CODE REVIEW

File: tests/torch/sota_checkpoints_eval.json [hunk 1]
```diff
@@ -250,7 +250,7 @@
                 "config": "examples/torch/classification/configs/binarization/resnet18_imagenet_binarization_xnor.json",
                 "reference": "resnet18_imagenet",
                 "target_ov": 61.82,
-                "target_pt": 61.74,
+                "target_pt": 61.88,
```
Comment (AlexanderDokuchaev) [inline tests/torch/sota_checkpoints_eval.json:253]: Measured metric for PT is not stable in result of using CUDA. Better increase target diff
```
              "diff_target_pt_min": -0.3,
              "diff_target_pt_max": 0.3
```

**Review by AlexanderDokuchaev — CHANGES_REQUESTED** [review]

File: tests/torch/sota_checkpoints_eval.json (hunk 1 above)
Comment (daniil-lyakhov) [reply tests/torch/sota_checkpoints_eval.json:253]: Agree, done

**Review by AlexanderDokuchaev — APPROVED** [review]

**Comment by daniil-lyakhov** [comment]
Build is green, @AlexanderDokuchaev, please merge

**Comment by MaximProshin** [comment]
@daniil-lyakhov @AlexanderDokuchaev , should it also be mapped to the release branch or is it only visible on the develop?

**Pull request merged**